examples/chart/teleport-kube-agent/templates/statefulset.yaml

```
@@ -113,9 +118,15 @@ spec:
         {{- if .Values.imagePullPolicy }}
         imagePullPolicy: {{ toYaml .Values.imagePullPolicy }}
         {{- end }}
-        {{- if .Values.extraEnv }}
+        {{- if or .Values.extraEnv .Values.tls.existingCASecretName }}
         env:
-          {{- toYaml .Values.extraEnv | nindent 10 }}
+        {{- if (gt (len .Values.extraEnv) 0) }}
+          {{- toYaml .Values.extraEnv | nindent 8 }}
```

```suggestion
          {{- toYaml .Values.extraEnv | nindent 10 }}
```

Sorry if it's not accurate, I'm still learning yaml :nerd_face:

---

YAML supports both
```yaml
env:
- var1
- var2
```
and
```yaml
env:
  - var1
  - var2
```

We seem to be alternating between both in all our files, sometimes even within the same file.

I think the comment is totally valid and is worth its own issue and PR to align all our YAML manifest at once. We'll have to choose a convention and follow it.

YAML is terrible.